src/main/java/ch/heigvd/pro/b04/votes/VoteIdentifier.java

```diff
@@ -0,0 +1,43 @@
+package ch.heigvd.pro.b04.votes;
+
+import ch.heigvd.pro.b04.answers.Answer;
+import ch.heigvd.pro.b04.participants.Participant;
+import ch.heigvd.pro.b04.participants.ParticipantIdentifier;
+import java.io.Serializable;
+import java.sql.Timestamp;
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import javax.persistence.ManyToOne;
+import javax.persistence.PrimaryKeyJoinColumn;
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+
+@Data
+@Embeddable
+public class VoteIdentifier implements Serializable {
+
+  @Column
+  private Timestamp timeVote;
+
+  @ManyToOne
+  @PrimaryKeyJoinColumn
```

Comment:
I'm not sure I understand what `@PrimaryKeyJoinColumn` does ?

Reply:
Me neither but I know it's necessarry